<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinesweeperClone</class>
 <widget class="QMainWindow" name="MinesweeperClone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinesSweeper Clone</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMinesweeperClone">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGame">
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollAreaGame">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="gameArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>673</width>
             <height>392</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="timersRefreshWidget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLCDNumber" name="lcdMines">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="refreshButton">
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdTime">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="gridButtons" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QGridLayout" name="gridLayout"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRankings">
       <attribute name="title">
        <string>Rankings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="scrollAreaRankings">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="rankingsArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>673</width>
             <height>392</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTabWidget" name="tabWidgetRankings">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabBeginner">
               <attribute name="title">
                <string>Beginner</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QScrollArea" name="scrollAreaBeginnerRanking">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="beginnerRankingArea">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>631</width>
                     <height>325</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tabIntermediate">
               <attribute name="title">
                <string>Intermediate</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QScrollArea" name="scrollAreaIntermediateRanking">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="intermediateRankingArea">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>631</width>
                     <height>325</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tabExpert">
               <attribute name="title">
                <string>Expert</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QScrollArea" name="scrollAreaExpertRanking">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="expertRankingArea">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>631</width>
                     <height>325</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_11"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSaveAs"/>
    <addaction name="actionLoad"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
